packages/cli-tools/bin/streamr-governance-vote.ts

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env node
+import pkg from '../package.json'
+import { createCommand } from '../src/command'
+import { getClientConfig } from '../src/client'
+import snapshot from '@snapshot-labs/snapshot.js'
+import { Wallet } from '@ethersproject/wallet'
+
+const hub = 'https://hub.snapshot.org'
+const snapshotClient = new snapshot.Client712(hub)
+
+const vote = async (privateKey: string, proposal: string, choice: number) => {
+    const wallet = new Wallet(privateKey)
+    try {
+        await snapshotClient.vote(wallet, wallet.address, {
+            space: 'streamr.eth',
+            proposal,
+            type: 'single-choice', // support only this type for now
+            choice,
```

Comment:
Does the `snapshotClient` properly deal with `choice` being `NaN` if `parseInt` previously was given some non-number?

Reply:
I tested and discovered that `snapshotClient` accepts a `choice: string` as well. Internally it attempts to create a `BigNumber` out of whatever is passed. 

So actually just passing in `choice` as a string results in an ok-ish error message for invalid inputs:

```
Error: invalid BigNumber string (argument="value", value="foo", code=INVALID_ARGUMENT, version=bignumber/5.7.0)
    at Logger.makeError (/Users/henri/work/network-monorepo/node_modules/@ethersproject/logger/lib/index.js:238:21)
    at Logger.throwError (/Users/henri/work/network-monorepo/node_modules/@ethersproject/logger/lib/index.js:247:20)
    at Logger.throwArgumentError (/Users/henri/work/network-monorepo/node_modules/@ethersproject/logger/lib/index.js:250:21)
    at Function.BigNumber.from (/Users/henri/work/network-monorepo/node_modules/@ethersproject/bignumber/lib/bignumber.js:199:27)
    at /Users/henri/work/network-monorepo/node_modules/@ethersproject/hash/lib/typed-data.js:125:47
    at /Users/henri/work/network-monorepo/node_modules/@ethersproject/hash/lib/typed-data.js:293:56
    at Array.map (<anonymous>)
    at /Users/henri/work/network-monorepo/node_modules/@ethersproject/hash/lib/typed-data.js:291:37
    at TypedDataEncoder.encodeData (/Users/henri/work/network-monorepo/node_modules/@ethersproject/hash/lib/typed-data.js:313:37)
    at TypedDataEncoder.hashStruct (/Users/henri/work/network-monorepo/node_modules/@ethersproject/hash/lib/typed-data.js:316:48) {
  reason: 'invalid BigNumber string',
  code: 'INVALID_ARGUMENT',
  argument: 'value',
  value: 'foo'
}
```

Do you think that's ok given the (semi-tech-literate?) user group of the cli tool, or should we go beyond this and add explicit validation of the input?